model/src/main/proto/onboarding.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
```

Comment:
why do we need this here?

Reply:
Removed.

Reply:
@nikitamarysolomanpvt Actually, I was working on app module and remove this line was creating issues. Please undo this change and add this line again.

Reply:
Done.